src/sst/elements/memHierarchy/membackend/backing.h

```diff
@@ -191,15 +191,15 @@ class BackingMalloc : public Backing {
         if (!fp) throw 1;
 
         size_t buffer_size;
-        fread(&buffer_size, sizeof(size_t), 1, fp);
-        fread(&alloc_unit_, sizeof(unsigned int), 1, fp);
-        fread(&shift_, sizeof(unsigned int), 1, fp);
-        fread(&init_, sizeof(bool), 1, fp);
+        (void) !fread(&buffer_size, sizeof(size_t), 1, fp);
```

Comment:
I understand casting to void to ignore the result, but why the ! ?

Reply:
> I understand casting to void to ignore the result, but why the ! ?

See the original PR comments at top.